tutorials/appengine-yarn/index.md

```diff
@@ -1,34 +1,39 @@
 ---
-title: Using yarn on App Engine
-description: Learn how to use yarn to manage dependencies for node.js applications on Google App Engine
+title: Using Yarn on App Engine
+description: Learn how to use Yarn to manage dependencies for Node.js applications on Google App Engine flexible environment.
```

Comment:
I very specifically don't want to include "flexible environment" everywhere in these docs.  We should chat about this.  There is no nodejs in standard, and if we *do* nodejs in standard, it will have the same support as flex.  Let's just cut these out.